lib/genevalidator/validation_alignment.rb

```diff
@@ -222,12 +219,11 @@ def run(n = 10)
     # Output:
     # Array of +String+s, corresponding to the multiple aligned sequences
     # the prediction is the last sequence in the vector
-    def multiple_align_mafft(prediction = @prediction, hits = @hits)
-      fail unless prediction.is_a?(Query) && hits[0].is_a?(Query)
+    def multiple_align_mafft(prediction, hits)
+      raise unless prediction.is_a?(Query) && hits[0].is_a?(Query)
 
-      options = ['--maxiterate', '1000', '--localpair', '--anysymbol',
-                 '--quiet', '--thread', @num_threads.to_s]
```

Comment:
Do you have statistics on how this changes the runtime?

Reply:
Not yet - this PR is Work in Progress (WIP) and so will try and put something together....


However, if this increases the run time, I think this is the right way to do this (otherwise we end up requesting a lot more threads than asked for) - which potentially causes things to break completely

Reply:
This can potentially decrease run time as well (running 40 x 40 = 1600 threads on 40 cores may incur considerable overhead in switching thread states). If the improvement can be quantified, it can be mentioned in the changelog (which is ++ for users). 

There are potentially better ways to do this still, for e.g., running 10 MAFFT processes each using 4 threads (that is, assuming the number of threads was set to 40), or 5 processes with 8 threads each, and so on (any m x n combination).